src/main/kotlin/uk/gov/justice/hmpps/offendersearch/services/SearchOperations.kt

```diff
@@ -76,4 +76,11 @@ fun BoolQueryBuilder.mustKeyword(value: Any?, query: String): BoolQueryBuilder {
     return this.must(QueryBuilders.matchQuery(query, value).analyzer("keyword"))
   }
   return this
+}
+
+
+fun BoolQueryBuilder.mustMatchOneOf(query: String, values: List<Any>): BoolQueryBuilder {
```

Comment:
Is this just to wrap should?  Makes much more sense to me!

Reply:
yeah, but with this list. Makes me wonder if someone has written a nice Elastic Search Kotlin extension library - I should check